src/api/product-sorting.ts

```diff
@@ -1,10 +1,20 @@
+import { useSearchParams } from '@remix-run/react';
 import { products } from '@wix/stores';
+import { useMemo } from 'react';
 import { ProductSortBy } from './types';
 
 export const SORT_BY_SEARCH_PARAM = 'sortBy';
 
 export const DEFAULT_SORT_BY = ProductSortBy.newest;
 
+export function useAppliedProductSorting() {
```

Comment:
I think that word `apply` here is redundant. surely every sorting, which is applied, is applied